### PR TITLE
Slider component

### PR DESCRIPTION
**Pull request by vinihvc**
image

### CODE REVIEW

File: apps/docs/docs/components/forms/slider.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+```jsx
+import { Slider } from '@blockadegames/pizza-juice';
+```
+
+## Usage
+
+Sliders reflect a range of values along a bar, from which users may select a single value. They are ideal for adjusting settings such as volume, brightness, or applying image filters
+
+The `useMediaQuery` hook returns an boolean, indicating whether the given query matches or queries match.
```
Comment (miguel-nascimento) [inline apps/docs/docs/components/forms/slider.mdx:18]: wtf?

File: apps/docs/docs/components/forms/slider.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <Slider orientation="vertical" css={{ minH: 200 }} />
+</Stack>
+
+---
+
+```jsx
+<Slider orientation="vertical" css={{ minH: 200 }} />
+```
+
+## Custom
```
Comment (miguel-nascimento) [inline apps/docs/docs/components/forms/slider.mdx:48]: It would be nice to add the multiple example too

**Review by miguel-nascimento — CHANGES_REQUESTED** [review]
Requesting change in the slider doc!

File: apps/docs/docs/components/forms/slider.mdx (hunk 1 above)
Comment (vinihvc) [reply apps/docs/docs/components/forms/slider.mdx:18]: lol

**Review by miguel-nascimento — APPROVED** [review]

**Pull request merged**